### PR TITLE
[FIZZ-45] Implement start menu

### LINKED ISSUE #45: Implement Start Menu
- [ ] Add separate scene for Start screen
- [ ] Add Start button that loads into the current scene
- [ ] Add Quit button that exits the game

### PR DESCRIPTION
**Pull request by Tirivitive**
Resolves #45

### CODE REVIEW

File: project-fizz/Assets/TextMesh Pro.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: f54d1bd14bd3ca042bd867b519fee8cc
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (NDGogo) [inline project-fizz/Assets/TextMesh Pro.meta:8]: Do we need an extra step to install TextMesh Pro? Just curious as when I run the game, the text doesn't fit in the Game window

File: project-fizz/Assets/Scripts/UI/StartScreenUIController.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class StartScreenUIController: MonoBehaviour
+{   
+    public void startGame()
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
+    }
+    public void quitGame()
```
Comment (NDGogo) [inline project-fizz/Assets/Scripts/UI/StartScreenUIController.cs:12]: New line between here pls

**Review by NDGogo — COMMENTED** [review]
Just double checking if the StartMenu scene has been set to the default scene to load?

**Comment by jjbdlsantos** [comment]
Note: when testing UI elements ensure display settings are set to:
- Full HD (1920 x 1080)
- Scale 1x

**Review by NDGogo — APPROVED** [review]

**Pull request merged**